### PR TITLE
Add mirrored image for gatekeeper 3.6.0 and gatekeeper-crds 3.6.0

### PR DESCRIPTION
**Pull request by rayandas**
#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in the images-list file
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexicographically)
- [x] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [ ] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

New image, version bump.

#### Linked Issues ####

https://github.com/rancher/rancher/issues/34593
#### Final Checks after the PR is merged ####
- [ ] Confirm that you can pull the new images and tags from DockerHub

### CODE REVIEW

File: images-list [hunk 1]
```diff
@@ -311,10 +311,12 @@ openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.3.0
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.4.0
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.5.1
 openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.5.2
+openpolicyagent/gatekeeper rancher/mirrored-openpolicyagent-gatekeeper v3.6.0
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.1.0
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.1.1
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.2.1
 openpolicyagent/gatekeeper rancher/openpolicyagent-gatekeeper v3.3.0
+openpolicyagent/gatekeeper-crds rancher/mirrored-openpolicyagent-gatekeeper-crds v3.6.0
```
Comment (cbron) [inline images-list:319]: @PennyScissors do we need the crds here ? we haven't mirrored that before.

Comment (rayandas) [reply images-list:319]: Gatekeeper upstream started adding crd image in `values.yaml` from `v3.6.0` you can find it [here](https://github.com/open-policy-agent/gatekeeper/blob/19ee221352a4d34ea659512e5ab376bae5fc17b8/charts/gatekeeper/values.yaml#L28).
In earlier versions there was only one image. For example if you check it for `v3.5.1` there was only the gatekeeper image. It's the [permalink](https://github.com/open-policy-agent/gatekeeper/blob/c8621550a3cc38d11f08b33a5c2b74a9ad328b1f/charts/gatekeeper/values.yaml#L24) for reference.

Comment (rayandas) [reply images-list:319]: Also a new repo has been created for crds. Here is the issue. https://github.com/rancherlabs/eio/issues/563

Comment (PennyScissors) [reply images-list:319]: @cbron That's correct, upstream added this new image in v3.6.0 so we need to mirror it.
image

**Review by cbron — APPROVED** [review]
Repo exists so we are ok

**Pull request merged**